ghost/admin/app/utils/member-filters/subscription-attribution.js

```diff
@@ -0,0 +1,21 @@
+import {MATCH_RELATION_OPTIONS} from './relation-options';
+
+export const SUBSCRIPTION_ATTRIBUTION_FILTER = {
+    label: 'Subscription started on post/page', 
+    name: 'conversion', 
+    valueType: 'string', 
+    resource: 'post', 
+    feature: 'memberAttribution', 
```

Comment:
Todo: remove this feature flag, and replace it with settings checks to enable/disable certain filters if certain features (not flags) are disabled